rackcity/views/change_plan_views.py

```diff
@@ -416,15 +416,17 @@ def change_plan_execute(request, id):
         ~Q(model__model_type=ModelType.BLADE_ASSET.value)
     )
     for asset_cp in rackmount_assets_cp:
-        changes = get_changes_on_asset(asset_cp.related_asset, asset_cp)
         updated_asset, created = get_updated_asset(asset_cp)
         updated_asset_mappings[asset_cp] = updated_asset
         if created:
             num_created += 1
             log_action(
                 request.user, updated_asset, Action.CREATE, change_plan=change_plan,
             )
+            asset_cp.differs_from_live = True
+            asset_cp.save()
```

Comment:
we also need to mark this as true so that the modification gets  returned after change plan is executed